lib/manager/npm/engines.js

```diff
@@ -0,0 +1,41 @@
+const semver = require('semver');
+const { getRepoReleases, semverSort } = require('../bazel/package');
```

Comment:
`bazel` seems like an odd dependency to have in an `npm` upgrade feature.

Reply:
yes, some refactoring needs to be done to push this github functionality into something generic. I may do that before merging this PR.